cranelift-codegen/meta/src/isa/x86/encodings.rs

```diff
@@ -1755,7 +1757,7 @@ pub fn define(
     extractlane_mapping.insert(32, (vec![0x66, 0x0f, 0x3a, 0x16], Some(use_sse41_simd))); // PEXTRD
     extractlane_mapping.insert(64, (vec![0x66, 0x0f, 0x3a, 0x16], Some(use_sse41_simd))); // PEXTRQ, only x86_64
 
-    for ty in ValueType::all_lane_types() {
+    for ty in ValueType::all_lane_types().filter(allowed_simd_type) {
```

Comment:
I assume it was unintentional to support b1 here.

Reply:
cc @abrown

Reply:
Correct, I think I caught it in most places but may have missed it here.